webapp/src/components/call_widget/component.tsx

```diff
@@ -452,6 +452,11 @@ export default class CallWidget extends React.PureComponent<Props, State> {
             return;
         }
 
+        // This is needed to prevent a conflict with the accessibility controller on buttons.
+        if (document.activeElement instanceof HTMLElement) {
+            document.activeElement.blur();
+        }
```

Comment:
Apologies, I don't fully understand why we're blurring the activeElement.. If you've already clicked it, or used the shortcut to 'click' it, then what does blurring it do?

Reply:
@cpoile When you click to unmute (or any button) the browser implicitly focuses on the button after which <kbd>Space</kbd> can be used to toggle it on/off. 

To reproduce:

- Start call.
- Unmute by clicking button.
- Try to mute back with shortcut. Pressing a single <kbd>Space</kbd> should work instead.

This is because it triggers the accessibility controller before we get a chance to do anything on our end. Unfocusing on click prevents this behaviour.

Reply:
Ah, I get it now. Yep, I can see it. Thanks.